cmd/image-builder/upload_test.go

```diff
@@ -36,7 +36,7 @@ func (fa *fakeAwsUploader) Check(status io.Writer) error {
 	return nil
 }
 
-func (fa *fakeAwsUploader) UploadAndRegister(r io.Reader, status io.Writer) error {
+func (fa *fakeAwsUploader) UploadAndRegister(r io.Reader, uploadSize int64, status io.Writer) error {
```

Comment:
This signature is incorrect, in images it is: `UploadAndRegister(r io.Reader, uploadSize uint64, status io.Writer) error` with the size being `uint64`.